htdocs/lib/settings-dev.inc.php

```diff
@@ -105,7 +105,7 @@
 $sql_replacements['tmpdb'] = $tmpdbname;
 
 // safemode_zip-binary
-$safemode_zip = '/var/www/bin/phpzip.php';
+$safemode_zip = '/var/www/html/local/prodsys/phpzip.php';
```

Comment:
I think we change this during the relaunch anyway, so I would not invest that much time now. I reverted the original script and copy it to `bin` on provision. So this is a minimal change, which will not affect deployment but fixes this issue in the VM